libs/core/src/lib/panel/panel.component.ts

```diff
@@ -105,4 +122,23 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit {
         this.expanded = !this.expanded;
         this.expandedChange.emit(this.expanded);
     }
+
+    /** @hidden */
+    _getButtonIcon(): string {
+        return this.expanded
+            ? 'slim-arrow-down'
+            : this._rtl
+                ? 'slim-arrow-left'
+                : 'slim-arrow-right'
+    }
+
+    /** @hidden */
+    private _listenRtl(): void {
+        this._subscription.add(
+            this._rtlService.rtl.subscribe(rtl => {
```

Comment:
You are not checking for `_rtlService` (this is optional dependency).

Reply:
@salarenko thank you, fixed